nowy.js

```diff
@@ -0,0 +1,198 @@
+    document.getElementById('reset-button').style.display = 'none';
+    document.getElementById('split-button').style.display = 'block';
+    timerStatus = "started"
+
+} else {
+
+    window.clearInterval(timerInterval);
+    document.getElementById('start-button').style.display = 'block';
+    document.getElementById('pause-button').style.display = 'none';
+    document.getElementById('reset-button').style.display = 'block';
+    document.getElementById('split-button').style.display = 'none';
+    timerStatus = "stopped";
+}};
+
+
+resetBtn.addEventListener('click', timeAndDataReset);
+
+function timeAndDataReset() {
+    milliseconds = 0;
+    seconds = 0;
+    hours = 0;
+    minutes = 0;
+    displayTimer = document.querySelector('.timer').innerText = '00:00:00:00';
+
+    document.querySelector('.lp-num').innerHTML = '';
+    document.querySelector('.split-results').innerHTML = '';
+    document.querySelector('.total-time').innerHTML = '';
+    LapNumber = 0;
+    totalTimes = [];
+    intervalTimes = [];
+  };
+
+let totalTimes = [];
+let intervalTimes = [];
+let LapNumber = 0;
+
+const timeString = document.querySelector('.timer').innerText;
+
+function parseTimeToMilliseconds(timeString) {
+ 
+  const parts = timeString.split(':');
+  const hours = parseInt(parts[0]);
+  const minutes = parseInt(parts[1]);
+  const seconds = parseInt(parts[2]);
+  const milliseconds = parseInt(parts[3]);
+  
+  const totalMilliseconds = hours * 3600000 + minutes * 60000 + seconds * 1000 + milliseconds;
+  return totalMilliseconds;
+}
+function millisecondsToTimeFormat(milliseconds) {
+  const hours = Math.floor(milliseconds / 3600000);
+  milliseconds %= 3600000;
+  const minutes = Math.floor(milliseconds / 60000);
+  milliseconds %= 60000;
+  const seconds = Math.floor(milliseconds / 1000);
+  const remainingMilliseconds = milliseconds % 1000;
+
+  return (
+    hours.toString().padStart(2, '0') + ':' +
+    minutes.toString().padStart(2, '0') + ':' +
+    seconds.toString().padStart(2, '0') + ':' +
+    remainingMilliseconds.toString().padStart(3, '0')
+  );
+}
+
+splitBtn.addEventListener('click', totalTimer);
+
+function totalTimer() {
+  let totalResult = document.querySelector('.timer').innerText;
+  totalTimes.push(totalResult);
+
+  const resultsContainer = document.querySelector('.total-time');
+  const TotalTimeElement = document.createElement('div');
+  TotalTimeElement.innerText = totalResult;
+  resultsContainer.appendChild(TotalTimeElement);
+
+  
+    LapNumber++;
+    const lapContainer = document.querySelector('.lp-num');
+    const lapElement = document.createElement('p');
+    lapElement.innerText = LapNumber;
+    lapContainer.appendChild(lapElement);
+
+
+    const intervalContainer = document.querySelector('.split-results');
+    const intervalElement = document.createElement('div');
+
+
+    if (totalTimes.length === 1) {
+      let intervalResult = document.querySelector('.timer').innerText;
```

Comment:
You don\t need `let` here, because you don't overwrite this variable. Use `const` instead